pyscriptjs/rollup.config.js

```diff
@@ -47,14 +61,7 @@ export default {
     }),
     // This will make sure that examples will always get the latest build folder
     !production && copy({
```

Comment:
If we are managing file copy paths, let's remove the `!production` on line 63 limiting this to not-production

Reply:
Agreed and removed.